### DISCUSSION TITLE
[Support]: Frigate captured snapshot from another time

### DISCUSSION THREAD

**Discussion by klingon888** [blakeblackshear/frigate#13061] [General Support] [answered]
### Describe the problem you are having

I had an event which occurred at 10:30PM on 8/13. But the snapshot which was generated was from 14:20. The snapshot had a filename of "front_left-1723573241.602883-911x3s.jpg", which correlates to 14:20 timestamp. Whereas the file's system timestamp is correct at 8/13/2024 10:30PM. This has happened quite a few times. This was from a fresh install of v14, no conversions from previous versions.

Snapshot in /frigate/clips:
![image](https://github.com/user-attachments/assets/8e4a9416-76c6-41d1-be90-475f13233636)

System filename & timestamp:
![image](https://github.com/user-attachments/assets/cf1ac175-fd5a-4c20-94ac-44250aea01c6)

Snapshot in Home Assistant Media:
![image](https://github.com/user-attachments/assets/fa861ec9-1cf7-4ce6-a7e4-94e857cf537f)

### Steps to reproduce

NA

### Version

0.14.0-da913d8

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
None
```

### Operating system

HassOS

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Amcrest IP5M-T1277EW-AI

**Comment by NickM-27** [comment, marked as answer]
this is not a bug, events last as long as the objects are detected. This car started being detected at 10:30 but the best snapshot was found to be at 14:20
